test/Microsoft.ML.TestFramework/DataPipe/TestDataPipeBase.cs

```diff
@@ -111,6 +112,14 @@ protected static TextLoader.Arguments MakeIrisTextLoaderArgs()
             {
                 var schema = transformer.GetOutputSchema(data.Schema);
 
+                // If it's a row to row mapper, then the output name.
+                if (transformer.IsRowToRowMapper)
```

Comment:
>IsRowToRowMapper [](start = 32, length = 16)

How about adding `mustFail`'s for `GetRowToRowMapper` on invalid data?

Reply:
That's pretty good, let's do that. Also I see I screwed up the comments here.

---
In reply to: [219619342](https://github.com/dotnet/machinelearning/pull/973#discussion_r219619342) [](ancestors = 219619342)